src/modules/draft/draftApi.js

```diff
@@ -107,6 +108,24 @@ export const fetchCompetences = query =>
     resolveJsonOrRejectWithError,
   );
 
+export const fetchCompetenceTitle = async competenceCode => {
+  let url;
+  if (competenceCode[1] === 'E') {
+    url = grepUrl(`/kjerneelementer-lk20/${competenceCode}`);
+  } else if (competenceCode[1] === 'M') {
```

Comment:
Syns dette er slightly mer tydelig om vi gjør noe sånt:
```suggestion
  if (competenceCode.startsWith('KE')) {
    url = grepUrl(`/kjerneelementer-lk20/${competenceCode}`);
  } else if (competenceCode.startsWith('KM')) {
```